src/univalue/include/univalue_escapes.h

```diff
@@ -1,6 +1,5 @@
-// Automatically generated file. Do not modify.
-#ifndef BITCOIN_UNIVALUE_UNIVALUE_ESCAPES_H
-#define BITCOIN_UNIVALUE_UNIVALUE_ESCAPES_H
+#ifndef BITCOIN_UNIVALUE_INCLUDE_UNIVALUE_ESCAPES_H
+#define BITCOIN_UNIVALUE_INCLUDE_UNIVALUE_ESCAPES_H
```

Comment:
I guess moving the files is required for build system reasons?

If not, I'd also prefer to leave the location as-is, unless there is a reason to change it.

Reply:
It's required if we want to have src/univalue integrated / linted as-if it's part of of this codebase. Otherwise we'd have to retain exceptions to at least the include linter, which would be somewhat confusing given it's no-longer a subtree.